### ISSUE TITLE
🥇 Hacktoberfest: Add your native language translation for symbol files

### ISSUE THREAD

**Issue by btk** [btk/aac-native#2]
Hello!

This app is an open source project, I work on my free time. And the app is published in google play. It is totally free. Basically what this app does is; lists **bunch of symbols and people with autism use this app to communicate** with their surroundings. 

That symbols triggers **TTS on press**.

But since most of the people with autism can only understand their native language, it is really important for this app to support multiple languages.

If you are a bilingual and want to contribute to this app, you can submit your translations of this app's symbol data.

**How To Submit A Translation?**

- For this project
- Duplicate file `/data/card_en.json` with your language code (eg: `/data/card_es.json`)
- Start translating the cards.

**Only translate** the `title` and `phrase` properties, other properties are used for symbol images etc.
```
	{
		"type": "card",
		"title": "Grandparents",
		"slug": "grandparents",
		"parents": [
			"people"
		],
		"phrases": [
			{
				"symbol": "hand",
				"phrase": "I love my grandparents"
			},
			{
				"symbol": "hand",
				"phrase": "I want to see my grandparents"
			}
		]
	}
```

So the properties needed to be translated are these;

```
	{
		"type": "card",
		"title": "**TRANSLATE_THIS**",
		"slug": "grandparents",
		"parents": [
			"people"
		],
		"phrases": [
			{
				"symbol": "hand",
				"phrase": "**TRANSLATE_THIS**"
			},
			{
				"symbol": "hand",
				"phrase": "**TRANSLATE_THIS**"
			}
		]
	}
```

I translated English to Turkish and it usually takes about **10 minutes if you are fast at typing** :) Think of the people using this app will appreciate you! And also get your hacktoberfest credits in 👍 

Currently translated;
- English @btk (done) 
- Turkish @btk (done)
- Spanish ..
- German ..
- Other languages ..

If you would like to, write a comment and the language you are going to be translating and I add you to the list not to create any conflicts.

Google play link: https://play.google.com/store/apps/details?id=org.dreamoriented.leeloo&hl=en
I will soon publish it to app store also.

**Comment by rafa-acioly** [comment]
i'll do for Portuguese (Brazil) can i use `card_pt-br.json`?

**Comment by btk** [comment]
@rafa-acioly yes, sure; I think we also actually do others in that format tho, we can use like this;

card_en_US.json
card_pt_BR.json
card_tr_TR.json
card_en_AU.json
...

Thank you I'm adding you

**Comment by rafa-acioly** [comment]
@btk the first phrase is an affirmation and the second is a question?

example:

```
{
    "symbol": "hand",
    "phrase": "I don't want to drink water"
},
{
    "symbol": "hand",
    "phrase": "Do you want to drink water"
},
```

**Comment by btk** [comment]
@rafa-acioly in this case, yes but not usually. So using english version as blue print would be better.

**Comment by rafa-acioly** [comment]
@btk i was thinking to use a question mark at the end like: 
`Do you want to drink water`: `Você quer beber água?`

**Comment by btk** [comment]
@rafa-acioly yes it is totally ok. Actually in english version question mark is missing in mistake probably.

**Comment by codesue** [comment]
I'd like to do the Swedish translation.

**Comment by easydaniel** [comment]
I'd like to do the Chinese translation.

**Comment by darrylsepeda** [comment]
I'll do for Indonesian language

**Comment by aaronmak** [comment]
I can do the mandarin translation!

**Comment by romainwn** [comment]
Working on the french translation

**Comment by viettrung9012** [comment]
I can do Vietnamese translation :)

**Comment by oscodeproject** [comment]
Working on dutch translation

**Comment by Wachiwi** [comment]
Working on german translation

**Comment by btk** [comment]
Thank you so much everybody, I'm assigning;

- Swedish @codesue (translating)
- Indonesian @darrylsepeda  (translating)
- Mandarin @aaronmak  (translating)
- French @romainwn  (translating)
- Indonesian @darrylsepeda  (translating)
- Vietnamese @viettrung9012  (translating)
- Dutch @oscodeproject  (translating)
- German @Wachiwi  (translating)

Just a small detail, if you add the language code in the format of en_US on your pull request, it will be easier for me. I will be checking all pull requests and merging throughout the day. 

Thank you!

**Comment by verginer** [comment]
I could add Italian if that is not already in the works.

**Comment by rafa-acioly** [comment]
#9 
Portuguese (Brazilian) finished :tada:

**Comment by btk** [comment]
@verginer sure, I'm adding you. Thank you!

**Comment by egin10** [comment]
have this done for Indonesian language ?

**Comment by btk** [comment]
@egin10 yes, @darrylsepeda is currently doing it. Maybe you can check after he is done for any mistakes. Thanks for your interest!

**Comment by boccob** [comment]
I'll add Russian translation.

**Comment by egin10** [comment]
okey, i'll try it and may i do it for hacktoberfest ?

**Comment by btk** [comment]
@boccob assigned you to Russian, thank you!

**Comment by egin10** [comment]
i've correcting for indonesian language, please check it @btk

**Comment by jainnidhi703** [comment]
I can do Hindi translation.

**Comment by btk** [comment]
@jainnidhi703 Great, assigned you to Hindi, thank you!

**Comment by jainnidhi703** [comment]
https://github.com/btk/aac-native/pull/12 has the Hindi Translation file.

**Comment by btk** [comment]
Merged Hindi and French, thank you so much. Currently 3 languages are getting translated;

- Swedish @codesue (translating)
- German @Wachiwi (translating)
- Russian @boccob (translating)

If you guys are busy or decided not to contribute, you can mention me to un-assign you. Thank you everybody.

---

PS: We are still open for contributions from other languages.

**Comment by rootEnginear** [comment]
I'll do for Thai translation.

**Comment by btk** [comment]
Thank you guys, another update;

- Swedish @codesue (translating)
- Russian @boccob (translating)

Only 2 languages are remaining translated. I'm making sure to mention you guys, as I stated before if you are busy or not interested anymore on contributing, please contact me via comments or e-mail (on my github profile) so I can un-assign you.

**Comment by codesue** [comment]
I was planning on working on it over the weekend. Is that okay? If you need it sooner you can reassign it.

**Comment by btk** [comment]
@codesue Hey, it is totally ok. Thank you so much.

**Comment by boccob** [comment]
@btk Sorry for delay, added #19

**Comment by sophiekoonin** [comment]
Do you need any Spanish translation?

**Comment by btk** [comment]
@sophiekoonin yes it would be great, I'm adding you to Spanish. Thank you so much!

**Comment by sophiekoonin** [comment]
@btk spanish has both the informal and polite form of the 2nd person pronoun (te & usted). Which would you prefer I used?

**Comment by btk** [comment]
@sophiekoonin I'm not sure about it since I'm not really familiar with that concept, mostly kids will be using this game. That might help you decide but I leave you the choice.

**Comment by sophiekoonin** [comment]
@btk there is also a small problem of gender - Spanish has different adjective endings for male and female. So while a man might be _enfadado_, I would be _enfadada_ (angry). I have been putting _enfadado/a_ but I realise this may be confusing for kids. What should I use?

**Comment by btk** [comment]
@sophiekoonin Again I'm not really sure, we did use slash for some languages (i guess portuguese) but since there is a lot of words like that. You should use the male one. Because %80 percent of kids  with autism are boys.

That makes me think we need to implement a gender specific way for locale language data.

**Comment by sophiekoonin** [comment]
@btk  Something to think about for the future! 

I am also happy to proof-read the English one if you like, I spotted a couple of non-native expressions that sounded a bit strange.

**Comment by btk** [comment]
@sophiekoonin That would be great, I prepared the English one and I'm not a native speaker. Also some other contributors fixes some of the mistakes, but I guess there still some exist.

Would be great if you can check it out.

**Comment by nikitakorba** [comment]
I can try to translate this to ukrainian

**Comment by btk** [comment]
@nikitakorba That's great, I'm adding you. If you have any problem or question you can ask it here. 

Thanks.

**Comment by btk** [comment]
@rafa-acioly @aaronmak @romainwn @darrylsepeda @viettrung9012 @oscodeproject @Wachiwi  @verginer @boccob @utkarsh-shekhar @rootEnginear @hejty @sophiekoonin @nikitakorba

Hey everyone, thank you all for your contribution. Addition to that, another AAC project has a similar task for checking the machine translation. Since there is over 10 people with different native languages, it would be really helpful if you like to check their translations.

Link to issue: https://github.com/shayc/cboard/issues/89

PS: Also the mentioned repository also has Hacktoberfest 🥇  tag.

**Comment by wkillerud** [comment]
Hi! I can translate to Norwegian if you'd like.

**Comment by btk** [comment]
@wkillerud that's great I'm adding you!

**Comment by YonatanHanan** [comment]
Working on the hebrew translation.

**Comment by btk** [comment]
@YonatanHanan That's great!

**Comment by deepkawal** [comment]
I can do Punjabi and Hindi Translation

**Comment by btk** [comment]
@deepkawal That's great, I'm adding you to list, but Hindi is already done, you might just check it when you have time, adding you to Punjabi.

**Comment by BrunoM24** [comment]
I can do the portuguese translation

**Comment by johnyaucc** [comment]
Hello, may I help on a version for Hong Kong? i.e. Cantonese/Hong Kong/Traditional Chinese.

**Comment by apoeco** [comment]
I can do the Greek translation!

**Comment by btk** [comment]
@johnyaucc @apoeco that would be great guys, hit me up if you have any questions

**Comment by romainwn** [comment]
Hi @btk , maybe I'll get more success here, could you take a look at #16 ? I think it would be great if we all could see the work you have done on this app, on our phones with a link on the Google play store !

**Comment by btk** [comment]
@romainwn there is a google play link at the OP post of this issue.

**Comment by sisgandarli** [comment]
I can translate into Azerbaijani language.

**Comment by ialixandroae** [comment]
Hi. I will do it for romanian.

**Comment by btk** [comment]
@ialixandroae Thanks!

**Comment by sorija** [comment]
Hello, I will do polish:)

**Comment by btk** [comment]
@sorija thats great, looking forward for your pr.

**Comment by Tarektouati** [comment]
hey, i can add Arabic

**Comment by btk** [comment]
Hey @Tarektouati that would be a great addition. Looking forward to your PR.

Adding you to the "doing" list. 👍
